### PR TITLE
Review one of my past projects (Tribute Page)

### PR DESCRIPTION
**Pull request by saqibjavaiddev**
This pull request is used to code review my own past project from Microverse Admission Trial. Learning objectives are:

- Apply best practices in HTML code.
- Use semantic HTML tags.
- Use CSS selectors correctly.
- Use CSS box model.
- Perform a code review for yourself on a past project.

### CODE REVIEW

**Comment by saqibjavaiddev** [comment]
Hi @saqibjavaiddev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
- Please consider this review a chance to learn and improve your coding skills, because that is what you are here for.

## Highlights
- Good HTML structure
- Usage of semantic HTML tags.
- Good CSS.
- You documented your work professionally

## Required Changes ♻️
_Check the comments under the review._

**Comment by saqibjavaiddev** [comment]
@saqibjavaiddev The figurecaption is not looking good. I suggest you to some margin.
image

**Comment by saqibjavaiddev** [comment]
@saqibjavaiddev, the list also need some space. I suggest you to add margin for list items too.
image

**Comment by saqibjavaiddev** [comment]
@saqibjavaiddev, In the original tribute page there is border-radius. Please try adding some border-radius for head class in your code.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @saqibjavaiddev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._
- [ ] I would advise that you use inline-Github comments at least once as this is a requirement. [Read more on how to use inline comments here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ <body>
+   <div class="main">
+     <div class="head">
+       <h1>Dr. Norman Borlaug</h1>
+       <p>The man who saved a billion lives</p>
+     </div>
+   <figure class = "images">
+       <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+       <figcaption>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+   </figure>
```
Comment (saqibjavaiddev) [inline index.html:18]: @saqibjavaiddev, this image is breaking the layout on some screens. I suggest you fix it.

- Also add alt text for the image

**Review by saqibjavaiddev — COMMENTED** [review]
Hi @saqibjavaiddev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

**Comment by saqibjavaiddev** [comment]
Hi [Munsa1](https://github.com/Munsa1), I need your help. Will you please guide me how to automatically fix linter errors on vscode.

**Review by muneebulrehman — APPROVED** [review]
Hi @saqibjavaiddev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by saqibjavaiddev** [comment]
Hi [muneebulrehman](https://github.com/muneebulrehman), I need your help. Will you please guide me how to automatically fix linter errors on vscode.

**Pull request merged**